### ISSUE TITLE
Unable to upgrade Ubuntu package from 5.11.85-1 to 5.11.90-1

### ISSUE THREAD

**Issue by baby-gnu** [OpenNebula/one#4917]
**Description**

Testing the 5.12 beta, I'm not able to upgrade the package from `5.11.85-1` to `5.11.90-1` because of a packaging issue:

```
Unpacking opennebula-common (5.11.90-1) over (5.11.85-1) ...
dpkg: error processing archive /tmp/apt-dpkg-install-sOV4xF/25-opennebula-common_5.11.90-1_all.deb (--unpack):
 trying to overwrite '/usr/lib/tmpfiles.d/opennebula-node.conf', which is also in package opennebula-node 5.11.85-1
```

**To Reproduce**

On Ubuntu Focal:

1. Install the `5.11.85-1` package
2. Upgrade to `5.11.90-1`

**Expected behavior**

The upgrade should be done without troubles.

**Details**
 - Affected Component: [e.g. Sunstone, Scheduler or Storage]
 - Hypervisor: KVM
 - Version: 5.11.90-1

**Additional context**

If the file `/usr/lib/tmpfiles.d/opennebula-node.conf` should be provided by `opennebula-common` instead of `opennebula-node`, the package should declare the [proper relationship](https://www.debian.org/doc/debian-policy/ch-relationships.html#overwriting-files-and-replacing-packages-replaces)

For example, the `opennebula-common` should have:

```
Replaces: opennebula-node (<< 5.11.90)
Breaks: opennebula-node (<< 5.11.90)
```

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by baby-gnu** [comment]
As the file in `5.10` is provided by `opennebula-node` too, this issue will break any migration path from `5.10` to `5.12`.

Regards.

**Comment by vholer** [comment]
File moves among packages has happened already in 5.11.85 and common package properly breaks/replaces anything older. Unfortunately, Ubuntu 20.04 has left behind and fixed in 5.11.90, but the versions in breaks/replaces weren't shifted.

I have adjusted the breaks/replaces versions for next release, but even without them, it shouldn't affect upgrades from stable versions in any way.

Thanks for the report.
